file.go

```diff
@@ -123,12 +145,24 @@ func (f *File) GetSection(name string) (*Section, error) {
 	}
 
 	sec := f.sections[name]
-	if sec == nil {
+	if sec == nil || len(sec) == 0 {
 		return nil, fmt.Errorf("section '%s' does not exist", name)
 	}
+
 	return sec, nil
 }
 
+// GetSectionCount gives you the count of sections with the same name.
+func (f *File) GetSectionCount(name string) int {
```

Comment:
What's the use case of this method?

Reply:
It is used in DeleteSection and I thought that it may be a feature, someone else could also need. That's why I made it public. But as it's only a len on f.GetSections it would be also ok to remove it and put the code directly into DeleteSection, if you don't like it.

Reply:
Yeah, just embed in the `DeleteSection` SGTM.